src/components/include/cbuf.h

```diff
@@ -469,8 +483,8 @@ __cbuf_freelist_pop(unsigned long sz, unsigned int flag)
 		old      = *(volatile unsigned long long *)target;
 		cm       = (struct cbuf_meta *)ofake[0];
 		nfake[0] = (unsigned long)cm->next;
-		nfake[1] = ofake[1]+1;
-	} while(unlikely(!cos_dcas(target, old, update)));
+		nfake[1] = ofake[1] + 1;
+	} while (unlikely(!cos_dcas(target, old, update)));
 
 	if (unlikely(cm == fl)) return NULL;
```

Comment:
dont get why it is allowing this, but putting other single-line conditionals on a separate line...